tests/manual-test-cases/Group19-DRS-Disabled/19-4-DRS-Disabled-License-Features.md

```diff
@@ -2,31 +2,27 @@ Test 19-4 - DRS Disabled License Features
 =======
 
 # Purpose:
-To verify that the license and feature checks required for a ROBO Advanced environment are displayed and updated on VCH Admin.
+To verify that vic-machine create checks for the DRS setting and command flags specific to resource pools in a vSphere environment without DRS, such as a ROBO deployment. If a VCH is being installed in a DRS-disabled environment, vic-machine create should warn that DRS is disabled and mention that any resource pool options supplied in the command will be ignored as they are not applicable in this environment.
 
 # References:
 1. [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
 2. [Provide License and Feature Check](https://github.com/vmware/vic/issues/7277)
-3. [vic-admin to report on license and feature compliance](https://github.com/vmware/vic/issues/7276)
+3. [vic-machine to provide license and feature check](https://github.com/vmware/vic/issues/7275)
 
 # Environment:
 This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
-* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
-* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+* 1 PSC linking 2 VCs, each containing a 3-host cluster with VSAN enabled and DRS disabled, and a single host cluster.
```

Comment:
not sure why this test needs any of these features other than a simple cluster with DRS disabled

Reply:
Updated.